src/peergos/server/storage/JdbcTransactionStore.java

```diff
@@ -0,0 +1,103 @@
+package peergos.server.storage;
+
+import peergos.server.sql.*;
+import peergos.server.util.Logging;
+import peergos.shared.crypto.hash.*;
+import peergos.shared.io.ipfs.cid.*;
+import peergos.shared.io.ipfs.multihash.*;
+import peergos.shared.storage.*;
+
+import java.sql.*;
+import java.util.*;
+import java.util.logging.*;
+import java.util.stream.*;
+
+public class JdbcTransactionStore implements TransactionStore {
+	private static final Logger LOG = Logging.LOG();
+
+    private static final String INSERT_TRANSACTIONS_BLOCK = "INSERT INTO transactions (tid, owner, hash) VALUES(?, ?, ?);";
```

Comment:
`pending_transaction` would be a better name for this table imho.

Reply:
There isn't a non pending transaction.